src/index.tsx

```diff
@@ -150,6 +161,16 @@ export default class Carousel extends React.Component<CarouselProps, CarouselSta
   }
 
   renderControls() {
+    const BUTTON_STYLE = {
+      ...DEFAULT_BUTTONS_STYLE,
+      background: '#ccc',
+      ...this.props.dotStyle
+    }
+    const ACTIVE_BUTTON_STYLE = {
+      ...DEFAULT_BUTTONS_STYLE,
+      background: '#000',
+      ...this.props.activeDotStyle
+    }
```

Comment:
I want to declare this style only once.
or create a class method to get this style.